llm/llama-3/README.md

```diff
@@ -0,0 +1,340 @@
+<!-- $REMOVE -->
+# Scale Serving Llama-3 on Any Cloud or Kubernetes with SkyPilot
+<!-- $END_REMOVE -->
```

Comment:
Consider having a nice "Llama3 + SkyPilot" banner too, or an AI-generated image of 3 llamas?

Reply:
Added one. Thanks!